src/main/java/com/nvs/store/auth/AuthService.java

```diff
@@ -37,15 +36,11 @@ public AuthResponse register(RegisterRequest request) {
 
         var jwtToken = jwtService.generateToken(user);
 
-        return AuthResponse
-                .builder()
-                .authStatus("Successfully registered")
-                .jwtToken(jwtToken)
-                .build();
+        return ResponseEntity.status(CREATED).body(jwtToken);
 
     }
 
-    public AuthResponse login(LoginRequest request) {
```

Comment:
if this changes are from @anovosiolovGD previous commits, please rebase your branch on develop and force push